var/spack/repos/builtin/packages/votca-csg-tutorials/package.py

```diff
@@ -20,8 +20,6 @@ class VotcaCsgTutorials(CMakePackage):
     git      = "https://github.com/votca/csg-tutorials.git"
     maintainers = ['junghans']
 
-    version('master', branch='master')
```

Comment:
We should first deprecate these versions and remove them in a future Spack release, see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions

Reply:
Good point, will add `deprecated=True`.

Reply:
But I removed `master` as the code was removed from the branch and it won't build either way.

Reply:
I would still deprecate master/stable in case someone already has those branches checked out locally in their downloads cache.

Reply:
Hmm, I feel the chance that somebody trying to build a deprecated `master` and ending up with obscure error messages as the branch has no code on it, is much bigger than somebody trying to build from their downloads cache.

In the end I am happy to follow the packing guide, but in this case this seems like we are trying to help 0.1% that have a local cache and screwing over 99.9% that don't read the deprecation message.

Reply:
> don't read the deprecation message.

Note that the deprecation message is a prompt that asks you if you're sure you want to install this version despite it being deprecated, so it's a little harder to ignore. You can still miss it if you use `spack install --yes-to-all` though.

You're the maintainer of these packages, so I'll leave the final decision to you. These are more "best practices" than strict policies.

Reply:
Yeah, I know my users, so I will opt for dropping master ;-)